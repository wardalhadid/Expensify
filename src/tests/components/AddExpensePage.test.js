// import React from 'react';
// import {shallow} from 'enzyme';
// import {AddExpensePage} from '../../components/AddExpensePage';

test('should render addExpensePage correctly', () => {
  // const onSubmit = jest.fn();
  // const navigate = jest.fn();
  // const wrapper = shallow(<AddExpensePage onSubmit={onSubmit} />);
  // expect(wrapper).toMatchSnapshot();
});

test('should handle onSubmit', () => {
  // const onSubmit = jest.fn();
  // const navigate = jest.fn();
  // const wrapper = shallow(<AddExpensePage onSubmit={onSubmit} />);
  // wrapper.find('ExpenseForm').prop('onSubmit')();
})
