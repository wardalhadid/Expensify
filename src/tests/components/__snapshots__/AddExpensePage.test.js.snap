// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render addExpensePage correctly 1`] = `ShallowWrapper {}`;
